marlowe-playground-client/src/Marlowe/Holes.purs

```diff
@@ -834,7 +893,7 @@ data Contract
   = Close
   | Pay AccountId (Term Payee) (Term Token) (Term Value) (Term Contract)
   | If (Term Observation) (Term Contract) (Term Contract)
-  | When (Array (Term Case)) (TermWrapper Slot) (Term Contract)
+  | When (Array (Term Case)) (Term ExtendedTimeout) (Term Contract)
```

Comment:
Not 100% sure, but TermWrapper was used in very few places... by adding ExtendedTimeout here, is TermWrapper still needed?

Reply:
yes, please check. it was annoying to have.

Reply:
It seems it is still used by `Scale` and `UseValue`